### PR TITLE
Add pixpress

### PR DESCRIPTION
**Pull request by devarellin**
> To make a submission to the Dapps Page, please complete the task lists
> below; otherwise remove them.

Your Dapp:

- [x] connects to Valora via WalletConnect V1 or DAppKit;
- [x] must be mobile-first or mobile responsive;
- [x] is in one of the existing categories; and
- [x] includes a link to user documentation.

### CODE REVIEW

**Comment by MuckT** [comment]
Main needs to be merged into this branch to grab the latest changes.
1. I am able to connect to the app just fine; however, when attempting swap I am getting 'All assets needs to be approve before swap'. No prompt is being served in Valora to approve the assets and because of this I am unable to swap.
2. One nitpick is the mobile UI leaves a lot of whitespace to the right on the swap columns.

**Comment by devarellin** [comment]
@MuckT Hi, clicking the lock icon can approve the asset.

But it seems not that intuitive. Maybe I should modify it to a button with approve text?

**Comment by devarellin** [comment]
Hi we've updated the approval button to text, this would be more intuitive.
update

**Comment by devarellin** [comment]
Thanks for suggestion! this is because it's only one asset, sometimes people swap multiple assets in the same transaction and the rest space gets filled

> One nitpick is the mobile UI leaves a lot of whitespace to the right on the swap columns.

**Comment by devarellin** [comment]
Hi @MuckT we've updated our connector, now every transaction will lead to Valora confirmation prompt.

**Comment by MuckT** [comment]
@devarellin - please swap the names to match icons.

#### Screenshot

**Comment by devarellin** [comment]
Oops, fixed @MuckT please have a look

File: locales/base.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "resource": "Stake SOURCE to help collateralize credit lines for regenerative businesses on ReSource",
     "ethichub": "Lend affordable capital to unbanked farmers",
     "mezumo": "Buy tokens over time using a simple strategy called Dollar-Cost Averaging (DCA)",
-    "uniswap": "Swap tokens or earn yield by providing liquidity"
+    "uniswap": "Swap tokens or earn yield by providing liquidity",
+    "pixpress": "Swap any amount of any cryto assets with others peer-to-peer and earn rewards"
```
Comment (MuckT) [inline locales/base.json:44]: Spelling of crypto.

**Review by MuckT — CHANGES_REQUESTED** [review]
Spelling fix.

**Pull request merged**